contracts/NftInjectableTokenIdRolesUpdatableBaseURI.sol

```diff
@@ -0,0 +1,94 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
```

Comment:
can we use the latest version instead of floating?

Reply:
0.8.9 as other contracts

Reply:
I think yes, will give a try

Reply:
https://hardhat.org/reference/solidity-support.html - latest supported Solidity is 0.8.9 for now, 0.8.10 isn't supported by hardhat yet. So will set to 0.8.9 in order to not break infrastructure